mmv1/third_party/terraform/services/filestore/resource_filestore_instance_test.go

```diff
@@ -190,3 +190,92 @@ resource "google_filestore_instance" "instance" {
 }
 `, name)
 }
+
+func TestAccFilestoreInstance_deletionProtection_update(t *testing.T) {
+	t.Parallel()
+
+	name := fmt.Sprintf("tf-test-%d", acctest.RandInt(t))
+	location := "us-central1-a"
+	tier := "ZONAL"
+
+	deletionProtection := true
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckFilestoreInstanceDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccFilestoreInstance_deletionProtection_create(name, location, tier),
+			},
+			{
+				ResourceName:            "google_filestore_instance.instance",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"zone", "location"},
+			},
+			{
+				Config: testAccFilestoreInstance_deletionProtection_update(name, location, tier, deletionProtection),
+			},
+			{
```

Comment:
optional: We _could_ add an empty config here as a TestStep with ExpectError that we fail on deletion due to deletion protection, but that's really just exercising the API feature and we'll confirm that the value is set with the ImportState suggested above already. Feel free to dismiss, though.

Reply:
Added the importState suggested above.